### PR TITLE
Update Data API reference

### PR DESCRIPTION
**Pull request by mDuo13**
- Adds new methods that were in the README from the Data API repo
- Fixes some remaining camelCase field names
- other minor changes to sync w/ https://github.com/ripple/rippled-historical-database/pull/172

### CODE REVIEW

File: content/references/data-api.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ The Ripple Data API v2 replaces the Historical Database v1 and the [Charts API](
 [v2.1.0]: https://github.com/ripple/rippled-historical-database/releases/tag/v2.1.0
 [v2.2.0]: https://github.com/ripple/rippled-historical-database/releases/tag/v2.2.0
 [v2.3.0]: https://github.com/ripple/rippled-historical-database/releases/tag/v2.3.0
+[v2.3.2]: https://github.com/ripple/rippled-historical-database/releases/tag/v2.3.2
```
Comment (ryangyoung) [inline content/references/data-api.md:27]: No v2.3.1?

File: content/references/data-api.md [hunk 2]
```diff
@@ -87,6 +89,7 @@ Validation Network Methods:
 * [Get Validator Validations - `GET /v2/network/validators/{:pubkey}/validations`](#get-validator-validations)
 * [Get Single Validator Reports - `GET /v2/network/validators/{:pubkey}/reports`](#get-single-validator-reports)
 * [Get Daily Validator Reports - `GET /v2/network/validator_reports`](#get-daily-validator-reports)
+* [Get rippled Versions - `GET /v2/network/rippled_versions`](#get-rippled-versions)
```
Comment (ryangyoung) [inline content/references/data-api.md]: Should that rippled be `rippled`?

File: content/references/data-api.md [hunk 3]
```diff
@@ -1944,6 +1947,128 @@ Response:
...
+| `data.date` | String | Date at which this period was calculated. |
+| `data.total` | Number | Total XRP volume exchanged during the period. |
+| `data.period` | String | Name of the period queried. |
+
+Each object in the `components` array of the Volume Objects represent the volume of a single external market.  Not all fields will be present for each market, depending on availability.
+
+| Field  | Value  | Description |
+|--------|--------|-------------|
+| `source` | String | Domain name of the specific external market. |
+| `base_volume` | Number | Exchange volume in terms of the base currency (XRP) |
```
Comment (ryangyoung) [inline content/references/data-api.md]: Will this always be XRP? The meaning of the parens here isn't clear to me. Also, I think the description is missing a period (.).

Comment (ryangyoung) [reply content/references/data-api.md]: nitpick: Hmm, looks like all of the descriptions are inconsistent w/r/t to periods. We should probably add to all or remove from all.

File: content/references/data-api.md [hunk 4]
```diff
@@ -2331,6 +2456,107 @@ Response:
 
 
 
+
+## Get Fee Stats
```
Comment (ryangyoung) [inline content/references/data-api.md:2464]: How are these methods organized? Probably outside the scope of this PR but we might want to readdress this. I was expecting them to be listed in alpha order but got...something else.

File: content/references/data-api.md [hunk 5]
```diff
@@ -2331,6 +2456,107 @@ Response:
...
+[[Source]<br>](https://github.com/ripple/rippled-historical-database/blob/develop/api/routes/network/getFeeStats.js "Source")
+
+Returns snapshots of the metrics derived from rippled's fee command. _(New in [v2.3.2][])_
```
Comment (ryangyoung) [inline content/references/data-api.md]: > rippled's

Should be?

> `rippled`'s 

And should that fee be in code font as well?

**Review by ryangyoung — APPROVED** [review]
Found a few nitpicks but overall looks good!

File: content/references/data-api.md (hunk 1 above)
Comment (mDuo13) [reply content/references/data-api.md:27]: Correct, [there isn't a v2.3.1](https://github.com/ripple/rippled-historical-database/releases/).

File: content/references/data-api.md (hunk 4 above)
Comment (mDuo13) [reply content/references/data-api.md:2464]: They're partly organized by the URL of the method, but not completely.

I agree we should fix it in a separate task.

File: content/references/data-api.md (hunk 3 above)
Comment (mDuo13) [reply content/references/data-api.md]: I didn't write this one, but I suspect that this means the base currency is always XRP.

As for the periods, my latest commit should fix that.

**Pull request merged**